### DISCUSSION TITLE
中國的防火牆

### DISCUSSION THREAD

**Discussion by Chengbosco** [1101-GS4719-C/Python-Community-GS4719-C#1699] [General]
https://www.youtube.com/watch?v=AIOMcbk-UL8&t=217s
中國的網路防火牆就像是一堵高大的城牆，被稱為「中國防火長城」。這堵牆由各種技術和法律手段組成，目的是控制和監控國內的網路流量，以符合政府的要求。
這城牆有幾道防線，例如DNS阻斷，就像城門一樣，防火牆可以擋住你通往某些網站的路，方法之一是修改網站的地址簿，使得你無法通過域名訪問被封鎖的網站，牆會直接提供錯誤的網址，讓你連線錯誤。防火牆也可以封鎖特定IP地址範圍，讓你無法通過這些地址訪問特定的網站或服務。而且利用協議層過濾，就像城牆上的哨兵，防火牆可以檢查你使用的工具，例如VPN或代理工具，並阻止你使用這些工具繞過牆。
影片內也用簡單的圖示演示如何翻牆，還有實現的困難。

**Comment by Yanxxx7979** [comment]
引申討論，我個人覺得這道"牆"最初的用意確實是好的，在之前還未建立起防火牆之前，中國的盜版網站氾濫，政府為了有效防治人民受騙及網路安全，而建立起防火牆。然而隨著民治及意識的提升後，這道牆的功用似乎轉為操控輿論及"監視"的作用，包括網路完全實名制，進而監控言行，我就覺得不甚恰當

**Comment by uwje** [comment]
阻擋盜版網站的初衷雖然是好的，但應該保持在阻擋，而不是進一步的去追蹤使用者的使用IP，甚至在習近平上任後，更加嚴格的再審查及篩選境外資料。
另外中國的言論審查我認為是不好的，因為他限制人們可看到的內容，雖然他們也不注重人權，但也希望他們能適當地開放。
